spikeextractors/extractors/mcsh5recordingextractor/mcsh5recordingextractor.py

```diff
@@ -71,22 +71,30 @@ def get_traces(self, channel_ids=None, start_frame=None, end_frame=None):
         if end_frame is None:
             end_frame = self.get_num_frames()
         if channel_ids is None:
-            channel_idxs = self._channel_ids
+            channel_ids = self._channel_ids
```

Comment:
You said, no mix up between index and id before ?
:)

Reply:
It wa snot mixed up, just a different naming